tools/quickstart/quickstart/validator_account.py

```diff
@@ -171,3 +175,44 @@ def get_author_address(base_dir) -> str:
             return author_address_file.read()
     except FileNotFoundError:
         return get_validator_address(base_dir)
+
+
+def handle_legacy_validator_key_file(base_dir, chain_dir):
+    if validator_key_file_exists(base_dir, chain_dir):
+        return
+
+    legacy_key_file_paths = legacy_validator_key_paths(base_dir, chain_dir)
+    if len(legacy_key_file_paths) > 1:
+        error_message = fill(
+            f"There are multiple files that correspond to validator keys: {legacy_key_file_paths}. "
+            f"Please make sure there is at most one key file in {os.path.join(base_dir, KEY_DIR, chain_dir)} "
```

Comment:
I would write
There are multiple files that correspond to validator keys: {legacy_key_file_paths}. "
            f"Please make sure there is one key file in {os.path.join(base_dir, KEY_DIR, chain_dir)} called 'account.json', so we know which one to select for the bridge. "

Because having multiple key files should be fine as I said.